common/src/main/java/org/apache/nemo/common/punctuation/Latencymark.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.nemo.common.punctuation;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Latency mark is conveyor that has data for debugging.
+ * It is created only from source vertex and record the timestamp when it is created and taskId where it is created.
+ */
+public final class Latencymark implements Serializable {
+  private final String createdtaskId;
+  private String lastTaskId;
+  private final long timestamp;
+
+  /**
+   * @param taskId task id where it is created
+   * @param timestamp timestamp when it is created
+   */
+  public Latencymark(final String taskId, final long timestamp) {
+    this.createdtaskId = taskId;
+    this.timestamp = timestamp;
+    this.lastTaskId = "";
+  }
+
+  /**
+   * @return the latencymark timestamp
+   */
+  public long getTimestamp() {
+    return timestamp;
+  }
+
+  /**
+   * @return the task id where it is created
+   */
+  public String getCreatedtaskId() {
```

Comment:
I cannot find the usage of `getCreatedtaskId()`.

Reply:
This method is not explicitly used. In the process of storing metrics in the json format. It seems that the getter is automatically scanned and used, Because If I remove this method, createdTaskId field is not included in json file.

Reply:
Could you please let me know the code block?

Reply:
storing json file is start from [here](https://github.com/apache/incubator-nemo/blob/65440c1a9acc5fdc4b7705de61610499ed2f6b84/runtime/master/src/main/java/org/apache/nemo/runtime/master/metric/MetricStore.java#L217). But I don't know about internal process

Reply:
Sorry, I cannot find the usage of this code. Only `getPreviousTaskId()` is called.